githubmarkdownui/job.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from githubmarkdownui.emoji import Emoji
+
+
+@dataclass
+class Job:
```

Comment:
Will validations app be one of the cases where this model can be used?

Reply:
Designed this with CI in mind but am currently changing this after a discussion with JZ. If validations app benefits from something like this, could you explain the use case?

Reply:
Talked to Terrence offline, the Validations app can potentially use the table function instead. We can resolve this conversation.